doc/release-notes/4.17-release-notes.md

```diff
@@ -0,0 +1,91 @@
+# Dataverse 4.17
+
+This release brings new features, enhancements, and bug fixes to Dataverse. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Dataset Level Explore Tools
+
+Tools that integrate with Dataverse can now be launched from the dataset page! This makes it possible to develop and add tools that work across the entire dataset instead of single files. Tools to verify reproducibility and allow researchers to compute on an entire dataset will take advantage of this new infrastructure.
+
+### Performance Enhancements
+
+Dataverse now allows installation administrators to configure the session timeout for users. Setting this lower will release system resources as configured and will result in better performance throughout a dataverse installation. 
+
+## Major Use Cases
+
+Newly-supported use cases in this release include:
+
+- As a user, I can launch and utilize tools that allow me to work across the code, data, and other files in a dataset. 
+- As a developer, I can build external tools to verify reproducibility or allow computation.
+- As an installation administrator, I can make a quick configuration change to provide a better experience for my installation's users.
+
+## Notes for Dataverse Installation Administrators
+
+### Configurable User Session Timeout
+
+Idle session timeout for logged-in users has been made configurable in this release. 
+The default is now set to 8 hours (this is a change from the previous default value of 24 hours). 
+If you want to change it, set the setting :LoginSessionTimeout to the new value *in minutes*. 
+For example, to reduce the timeout to 4 hours:                                                                                                                                                                          
```

Comment:
If you add a newline here and indent the curl command to four characters I think it will look better.

Reply:
will do